### PR TITLE
Initial 1.19.3 port

### PR DESCRIPTION
**Pull request by Cypher121**
Math module disabled for now.

ItemGroups extensions disabled: switch to Fabric for now or wait until QSL comes through with Groups v2?

### CODE REVIEW

File: library/src/main/kotlin/org/quiltmc/qkl/library/brigadier/BrigadierDsl.kt [hunk 1]
```diff
@@ -57,11 +58,13 @@ public fun <S> CommandDispatcher<S>.register(
  *
  * @see CommandBuildContext
  */
+@Suppress("MISSING_DEPENDENCY_CLASS")
```
Comment (Cypher121) [inline library/src/main/kotlin/org/quiltmc/qkl/library/brigadier/BrigadierDsl.kt:61]: Note: don't release this until re-built against a mappings release QuiltMC/quilt-mappings#307

File: library/src/main/kotlin/org/quiltmc/qkl/library/client/screen/QuiltScreen.kt [hunk 2]
```diff
@@ -14,17 +14,16 @@
  * limitations under the License.
  */
 
-@file:Environment(EnvType.CLIENT)
+@file:ClientOnly
```
Comment (Cypher121) [inline library/src/main/kotlin/org/quiltmc/qkl/library/client/screen/QuiltScreen.kt:17]: This entire file gives warnings as extensions like `public val Screen.buttons get() = (this as QuiltScreen).buttons` are meaningless when QuiltScreen is already injected into Screen. Is this a left-over from before it was injected and should be removed?

**Comment by Cypher121** [comment]
Mappings updated! Also added .gitattributes, same as QSL and template mod

**Review by SilverAndro — APPROVED** [review]

**Comment by SilverAndro** [comment]
Personally id wait for quilt on item groups, we dont want to have to release another breaking version just to bring up to speed

**Review by Oliver-makes-code — APPROVED** [review]
LGTM

**Review by NoComment1105 — APPROVED** [review]
LGTM, I agree on waiting for item group stuff

**Review by PotatoPresident — APPROVED** [review]

**Pull request merged**